drivers/Web/template.py

```diff
@@ -1,21 +1,11 @@
 import os.path
+from jinja2 import FileSystemLoader, Environment
 
-from drivers.Web.lexer import lexer, auxiliar_lexer
```

Comment:
Move the lexer and parser out of this repository and remove all assets related to them.